oci/common_columns.go

```diff
@@ -14,3 +25,63 @@ const (
 	ColumnDescriptionDefinedTags  = "Defined tags for resource. Defined tags are set up in your tenancy by an administrator. Only users granted permission to work with the defined tags can apply them to resources."
 	ColumnDescriptionSystemTags   = "System tags for resource. System tags can be viewed by users, but can only be created by the system."
 )
+
+func commonColumnsForAllResource(columns []*plugin.Column) []*plugin.Column {
+	return append(columns, []*plugin.Column{
+		{
+			Name:        "tenant_name",
+			Type:        proto.ColumnType_STRING,
+			Hydrate:     getCommonColumns,
+			Description: ColumnDescriptionTenantName,
+			Transform:   transform.FromField("Name"),
+		},
+	}...)
+}
+
+// get columns which are returned with all tables: tenant id, tenant name
+func getCommonColumns(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+
+	// Trace logging to debug cache and execution flows
+	plugin.Logger(ctx).Trace("getCommonColumns", "status", "starting", "connection_name", d.Connection.Name)
```

Comment:
@ParthaI should we include the `Trace` log here? Shouldn't we update it to `Debug` or `Info`?

Reply:
I think `Debug` is best, `Trace` is probably better for CLI/SDK messages, `Debug` is helpful for plugins